en/docs/setup/analytics/configure-analytics-for-elk-stack.md

```diff
@@ -2,29 +2,51 @@
 
 ## Step 1 - Setup APK
 
-1. Start by following the instructions outlined in [Customize Configurations](../Customize-Configurations.md). These instructions will guide you through the process of acquiring the `values.yaml` file, which you will then use to tailor the analytics configurations to your specific needs. Open `values.yaml` file.
+1. Start by following the instructions outlined in [Customize Configurations](../Customize-Configurations.md). These instructions will guide you through the process of acquiring the `values.yaml` file.
+2. Open the `values.yaml` file, and add the above configuration to the gatewayRuntime section under dp.
 
-2. Set following config under `wso2.apk.dp.gatewayRuntime` section to enable analytics.
+```yaml
+gatewayRuntime:
+  analytics:
+    enabled: true
+    publishers:
+    - enabled: true
+      type: "elk"
+```
+
+!!! Note
+    Optionally, `logLevel` can be configured for ELK. By default, this config is set to `INFO`.
 
     ```yaml
-    analytics:
-      enabled: true
-      publishers:
-      - enabled: true
-        type: "elk"
+    gatewayRuntime:
+      analytics:
+        enabled: true
+        publishers:
+        - enabled: true
+          type: "elk"
+          logLevel: "INFO"
     ```
 
-    !!! Note
-        Optionally, `logLevel`. By default, this config is set to `INFO`.
+Your values.yaml file will have a structure as follows.
```

Comment:
Now the structure of your values.yaml file will be as follows:

Reply:
Fixed in https://github.com/wso2/docs-apk/pull/584